content/en/docs/rules/_index.md

```diff
@@ -48,38 +48,51 @@ Key | Required | Description | Default
 
 ## Conditions
 
-The key part of a rule is the _condition_ field. A condition is simply a Boolean predicate on Sysdig events expressed using the Sysdig [filter syntax](http://www.sysdig.com/wiki/sysdig-user-guide/#filtering). Any Sysdig filter is a valid Falco condition (with the exception of certain excluded system calls, discussed below). In addition, Falco conditions can contain macro terms (this capability is not present in Sysdig syntax).
+The key part of a rule is the _condition_ field. A condition is a Boolean predicate expressed using the [condition syntax](/docs/rules/conditions). It is possible to express conditions on [all supported events](/docs/rules/supported-events) using their respective [supported fields](/docs/rules/supported-fields).
 
 Here's an example of a condition that alerts whenever a bash shell is run inside a container:
 
 ```
 container.id != host and proc.name = bash
 ```
 
-The first clause checks that the event happened in a container (Sysdig events have a `container` field that is equal to `"host"` if the event happened on a regular host). The second clause checks that the process name is `bash`. Note that this condition does not even include a clause with a system call! It only checks event metadata. Because of that, if a bash shell does start up in a container, Falco outputs events for every syscall that is performed by that shell.
+The first clause checks that the event happened in a container (where `container.id` is equal to `"host"` if the event happened on a regular host). The second clause checks that the process name is `bash`. Since this condition does not include a clause with a system call it will only check event metadata. Because of that, if a bash shell does start up in a container, Falco outputs events for every syscall that is performed by that shell.
 
-A complete rule using the above condition might be:
+If we only want to be alerted for each successful spawn of a shell in a container we can add the appropriate event type and direction to our condition:
```

Comment:
If you want to be alerted only for each successful spawn of a shell in a container, add the appropriate event type and direction to our condition: